test/testenv/cmutil.go

```diff
@@ -107,24 +130,52 @@ func CheckSearchHeadRemoved(deployment *Deployment) bool {
 	return searchHeadRemoved
 }
 
-// ClusterMasterSitesResponse is a representation of the sites managed by a Splunk cluster-master
-// Endpoint: /services/cluster/master/sites
-type ClusterMasterSitesResponse struct {
-	Entries []ClusterMasterSitesEntry `json:"entry"`
-}
-
-// ClusterMasterSitesEntry represents a site of an indexer cluster with its metadata
-type ClusterMasterSitesEntry struct {
-	Name    string                    `json:"name"`
-	Content ClusterMasterSitesContent `json:"content"`
+// RollHotBuckets roll hot buckets in cluster
+func RollHotBuckets(deployment *Deployment) bool {
+	podName := fmt.Sprintf("splunk-%s-cluster-master-0", deployment.GetName())
+	stdin := "/opt/splunk/bin/splunk rolling-restart cluster-peers -auth admin:$(cat /mnt/splunk-secrets/password)"
```

Comment:
Could we just use the roll-hot-buckets command instead of rolling-restart?

Reply:
I will do that as part of next PR. I just added this util but its not being used. I am using roll-hot-buckets on standalone which is part of this PR and will follow similar pattern for Indexer cluster.